.pre-commit-config.yaml

```diff
@@ -94,6 +94,18 @@ repos:
         language: system
         files: .*.go
         pass_filenames: false
+      - id: go-test
```

Comment:
Shouldn't we be running all tests in the pre-commit? I get your argument that if what you changed shouldn't impact longer tests, don't run. But also it seems that we are removing part of the benefit of the tests by not running them consistently. I'm not always fully aware of the impact of my changes

Reply:
The tests are still going to run completely in the CI, they weren't being run previously on pre-commit. Running long tests in pre-commit is going to be too cumbersome and time-consuming for every commit.

Reply:
ok!